File: Mathe/Reihen/harm_Reihe.py
# Berechnung der harmonischen Reihe
import time

k = 1
s = 0.0

timeout = input('Für wie viele Sekunden soll das Skript ausgeführt werden?')   # [seconds]

timeout_start = time.time()

while time.time() < timeout_start + timeout:
    s_temp = 1.0/k
    k = k + 1
    s = s + s_temp
    print(s)

File: Mathe/Reihen/B-Ware/harm_reihe.c
double k = 1;
double s = 0;

int timeout = 10;   // [seconds]

// timeout_start = time.time()

while (timeout > 0) {
    double s_temp = 1 / k;
    double k++;
    double s += s_temp;
    printf(s);
}